packages/@o3r/core/schematics/application/index.ts

```diff
@@ -0,0 +1,56 @@
+import { apply, chain, externalSchematic, MergeStrategy, mergeWith, move, renameTemplateFiles, Rule, schematic, strings, template, url } from '@angular-devkit/schematics';
+import { getPackagesBaseRootFolder, getWorkspaceConfig } from '@o3r/schematics';
+import * as path from 'node:path';
+import { NgGenerateApplicationSchema } from './schema';
+import type { PackageJson } from 'type-fest';
+
+/**
+ * Add an Otter application to a monorepo
+ *
+ * @param options Schematic options
+ */
+export function generateApplication(options: NgGenerateApplicationSchema): Rule {
+  const cleanName = strings.dasherize(options.name);
+
+  const createAppPackageJson = (targetPath: string, rootDependencies: Record<string, string | undefined>) => {
```

Comment:
Actually the name is not really matching the implementation.
Today the `templates/` folder does contain only the `package.json` file but this function will take all the files of this folder, not only a `package.json`.
Maybe cleaner to rename it to a more generic manner (or target only the `package.json` file explicitly)

Reply:
how about `addProjectSpecificFiles`?